packages/suite/src/support/messages.ts

```diff
@@ -8792,4 +8792,37 @@ export default defineMessages({
         id: 'TR_STAKE_PROVIDED_BY',
         defaultMessage: 'Provided by',
     },
+    TR_STAKE_YOUR_FUNDS_MANAGED_1: {
+        id: 'TR_STAKE_YOUR_FUNDS_MANAGED_1',
+        defaultMessage: 'Your staked funds are',
+    },
+    TR_STAKE_YOUR_FUNDS_MANAGED_2: {
+        id: 'TR_STAKE_YOUR_FUNDS_MANAGED_2',
+        defaultMessage: 'managed by Everstake',
+    },
```

Comment:
Please merge those separated strings into one and use similar logic to e.g. translation `TR_MOBILE_APP_PROMO_TEXT_FOOTER`

```
  <Translation
      values={{ h: text => <HeadingAccent>{text}</HeadingAccent> }}
      id="TR_STAKE_YOUR_FUNDS_MANAGED"
  />

    TR_STAKE_YOUR_FUNDS_MANAGED: {
        id: 'TR_STAKE_YOUR_FUNDS_MANAGED',
        defaultMessage: 'Your staked funds are <h>managed by Everstake<h>',
    },
```

Reply:
@shotgunofdeath i will fix it